cloudproxy-ui/src/plugins/bootstrap-vue.js

```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue'
+
+import BootstrapVue from 'bootstrap-vue'

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/claffin/cloudproxy/pullRequest?prid=7329682)
```suggestion
import BootstrapVue from "bootstrap-vue"
```